src/main/java/jewellery/inventory/service/SaleService.java

```diff
@@ -40,43 +49,31 @@ public List<SaleResponseDto> getAllSales() {
   @LogCreateEvent(eventType = EventType.SALE_CREATE)
   @Transactional
   public SaleResponseDto createSale(SaleRequestDto saleRequestDto) {
+
+    throwExceptionWhenNoResourcesAndProductsInRequest(saleRequestDto);
```

Comment:
Let's keep the throw method names consistent -> throwExceptionIfNoResourcesAndProductsInRequest

Reply:
done!